timers.js

```diff
@@ -1,4 +1,92 @@
+/******************************************
+BONUS VERSION WITH JQUERY
+******************************************/
 
+// how do I change my global variables in init to local variables that everything in the timer object can access?
 
+var timer = {
+  init: function() {
+    stopWatch = $("#timer");
+    reset = $("#reset");
+    start = $("#start");
+    pause = $("#pause");
+    timerStatus = "off";
```

Comment:
Might make more sense to make `timerStatus` a boolean value. That way, when you check it's value later one, all you have to do is `if( timerStatus ){ ... }`.